docs/_install-and-build/Installing-Geth.md

```diff
@@ -3,310 +3,411 @@ title: Installing Geth
-bundle; running as a docker container; or building it yourself. This document details all
-of the possibilities to get you joining the Ethereum network using whatever means you
-prefer. A list of stable releases can be found [here][geth-releases].
+There are several ways to install Geth, including via a package manager, downloading a pre-built bundle, running as a docker container or building from downloaded source code. On this page the various installation options are explained for several major operating systems. Users prioritizing ease of installation should choose to use a package manager or prebuilt bundle. Users prioritizing customization should build from source. It is important to run the latest version of Geth because each release includes bugfixes and improvement over the previous versions. The stable releases are recommended for most users because they have been fully tested. A list of stable releases can be found [here][geth-releases]. Instructions for updating existing Geth installations are also provided in each section.
+
 
 {:toc}
 
 -   this will be removed by the toc
 
-## Updating Geth
-
-Updating go-ethereum is as easy as it gets. You just need to download and install the
-newer version of geth, shutdown your node and restart with the new software. Geth will
-automatically use the data of your old node and sync the latest blocks that were mined
-since you shutdown the old software.
+## Package managers
 
-## Install from a package manager
+### MacOS via Homebrew
 
-### Install on macOS via Homebrew
+The easiest way to install go-ethereum is to use the Geth Homebrew tap. The first step is to check that Homebrew is installed. The following command should return a version number.
 
-The easiest way to install go-ethereum is to use our Homebrew tap. If you don't have
-Homebrew, [install it first][brew].
+<br>
+```shell
+brew -v
+```
+<br>
 
-Run the following commands to add the tap and install `geth`:
+If a version number is returned, then Homebrew is installed. If not, Homebrew can be installed by following the instructions [here][brew]. With Homebrew installed, the following commands add the Geth tap and install Geth:
 
+<br>
 ```shell
 brew tap ethereum/ethereum
 brew install ethereum
 ```
+<br>
 
-You can install the master branch using the `--devel` parameter:
 
+The previous command installs the latest stable release. Developers that wish to install the most up-to-date version can install the Geth repository's master branch by adding the `--devel` parameter to the install command:
+
+<br>
 ```shell
 brew install ethereum --devel
 ```
+<br>
+
+These commands install the core Geth software and the following developer tools: `clef`, `devp2p`, `abigen`, `bootnode`, `evm`, `rlpdump` and `puppeth`. The binaries for each of these tools are saved in `/usr/local/bin/`. The full list of command line options can be viewed in the terminal by running `geth --help`.
```

Comment:
Just out of curiosity, where did you get this list? check the binaries installed?

Reply:
This list came from the Executables section in the main Geth repository (which I assume to be more up to date than the website docs): https://github.com/ethereum/go-ethereum#executables